package.json

```diff
@@ -75,6 +75,9 @@
         "react-beautiful-dnd": "^10.1.1",
         "resize-observer-polyfill": "^1.5.1"
     },
+    "resolutions": {
```

Comment:
Is this a temporary (remove before merge) or permanent solution?

Reply:
> Is this a temporary (remove before merge) or permanent solution?

I guess it's temporary until `cli-app-scripts` gets its version bumped to a major 9...